interface/presenter/user_presenter.go

```diff
@@ -0,0 +1,27 @@
+package presenter
+
+import "github.com/alexis-aguirre/golang-bootcamp-2020/domain/model"
+
+type userPresenter struct {
+}
+
+type UserPresenter interface {
+	ResponseUsers(us []*model.User) []*model.User
+	ResponseUser(us *model.User) *model.User
+}
+
+//NewUserPresenter generates a new instance of UserPresenter
+func NewUserPresenter() UserPresenter {
+	return &userPresenter{}
+}
+
+func (up *userPresenter) ResponseUsers(us []*model.User) []*model.User {
+	for _, u := range us {
+		u.Name = "Mr. " + u.Name
```

Comment:
This is a bit far-fetched, but I think users could have an additional field for `Title`, that way you can be a bit more inclusive when presenting the information.